docs/components/list.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Compact UI | List</title>

```

Comment:
do the naming int he format List | compact UI as the user will need to see the component's name first in the tab

Reply:
Okay, will implement it.